### DISCUSSION TITLE
Upgrade keepassxc from 2.6.6 to 2.7.4 on Ubuntu

### DISCUSSION THREAD

**Discussion by lucabarbetti** [keepassxreboot/keepassxc#9113] [Q&A] [answered]
I have KeepassXC 2.6.6 (installed with "apt install" from the Ubuntu repository), what is the recommended "path" to follow in order to upgrade from 2.6.6 to the latest release 2.7.4?
Thank you

**Comment by droidmonkey** [comment, marked as answer]
Use flatpak distribution and uninstall from apt.

**Reply by lucabarbetti** [reply]
Thank you, droidmonkey... I followed your advice and now I have 2.7.4 (Flatpak). 
Thank you!

**Reply by droidmonkey** [reply]
Great you're welcome!
